### DISCUSSION TITLE
Why is not my repository showing up in the list?

### DISCUSSION THREAD

**Discussion by ybjbox** [community/community#131892] [Repositories]
These two repositories are not mine, why are they displayed in my list of repositories and cannot be deleted.![image](https://github.com/user-attachments/assets/332a8818-a800-4453-9b57-2d1340bfa869)

**Comment by bicced** [comment]
This is a search for repositories, not just yours. Don't worry about it, those repositories are not on your account.

**Comment by TheGiraffe3** [comment]
Because you have contributed to those repositories, you have a link to them.
